src/components/CardStatus/CardStatus.js

```diff
@@ -1,48 +1,65 @@
-import React, {useState, useEffect} from 'react'
-import './CardStatus.css'
+import React, { useState, useEffect } from "react";
+import "./CardStatus.css";
 
-export  const CardStatus = (props) => {
+/**
+ * 
+ *  I will write in this component but it apply to a lot of components
+    In component like this one you set redoundant statuses and for that
+    you needed to write effect to set thenm. This force react to do 
+    un-necessary re-render of the inner elements. 
+    it's better to use just the props.
+    States and effect need to be use when you have external interactivity 
+    from the props. in this case is not make a lot of sense. 
+    If you need transformation maybe it's better to use a useMemo or a useCallback.  
+ */
```

Comment:
@huvber Makes total sense. Thank you for that specifically, the example you gave in the CardAction component is a brutal piece of learning. 

That was the way my brain currently thinks react. However the example you shared made me see how different it can be.